scripts/core/EXT.rst

```diff
@@ -14,25 +18,48 @@ Objective
 =========
 
 Features which are device- or vendor-specific can be exposed as extensions.
-The list of extensions supported by the driver implementation can be queried using ${x}DriverGetExtensionProperties.
 
 There are two types of extensions defined by this specification:
 
 1. **Standard** - extensions ratified for inclusion into the current and all future versions of the specification.
 2. **Experimental** - extensions require additional experimentation and feedback from application vendors
    before ratification, therefore applications should not rely on experimental extensions in production.
 
-In addition, an implementation may choose to provide additional non-standard extensions via ${x}DriverGetExtensionFunctionAddress.
-It is the responsibility of the implementation to ensure these extensions do not conflict or interfere with any features or extensions of the core APIs.
-These extensions are not documented as part of this specification.
+In addition to above, an implementation may choose to provide non-standard extensions which are
+not defined in this specification.
+
+It is the responsibility of the implementation to define and document such non-standard extensions
+and also to ensure these extensions do not conflict or interfere with any features or extensions of the standard 
+core or sysman APIs.
```

Comment:
core, sysman or tools

Reply:
since tools doesn't have driver handle, restricting this PR to core and sysman only for now and we can revisit tools extension discovery separately